src/Resources/config/controllers.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service public="true" id="sonata.classification.controller.category_admin" class="Sonata\ClassificationBundle\Controller\CategoryAdminController">
+            <tag name="container.service_subscriber"/>
+            <call method="setContainer">
+                <argument type="service" id="Symfony\Component\DependencyInjection\ContainerInterface"/>
```

Comment:
this should be `Psr\Container\ContainerInterface`, otherwise we are injecting the full container.

Reply:
Will try: https://github.com/sonata-project/SonataClassificationBundle/pull/680
But seems like it's not so simple